Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreter.cs

```diff
@@ -195,7 +195,7 @@ class AstPythonInterpreter : IPythonInterpreter, IModuleContext, ICanFindModuleM
             return names.MaybeEnumerate().ToArray();
         }
 
-        public IPythonModule ImportModule(string name) {
+        public async Task<IPythonModule> ImportModuleAsync(string name) {
```

Comment:
Let's add `CancellationToken` on this level as well and pass it to the `TryImportModuleAsync`

Reply:
Nevermind, we use `CancellationTokenSource` below, mixing it with another `CancellationToken` requires too much code changes.

Reply:
On the other hand, it is public API, so we may want to do it right.

Reply:
Added, change propagated all over...